lib/CGenParser_lib.ml

```diff
@@ -148,10 +148,10 @@ module Do
 			    (MiscParser.Location_reg (p,loc),
 			     (ty,ParsedConstant.nameToV loc))::env)
 			   env t.params)
-        init prog_litmus in
-    let procs = List.map (fun p -> p.CAst.proc) prog in
+        init (fst prog_litmus) in
```

Comment:
It would be clearer to change the `let` bindings of lines 132 and 134 just above into:
```OCaml
    let prog,data =
      I.call_parser_loc "prog" chan prog_loc L.deep_lexer L.deep_parser in
    let prog_litmus,data_litmus =
      I.call_parser_loc "prog_litmus" chan prog_loc L.shallow_lexer L.shallow_parser in
```
That way we'd avoid using `fst` and `snd` and have more explicit variable names.